prow/cmd/crier/main.go

```diff
@@ -78,21 +86,44 @@ func (o *options) Validate() error {
 		}
 	}
 
+	if o.githubWorkers > 0 {
+		// no dry-run mode for crier
```

Comment:
Why not? I think it would be pretty useful to be able to run this in dry-run mode while debugging or to validate a change works as intended.

Reply:
added dryrun flag, but effective only for github now. Need to sort out how to dryrun gerrit/pubsub client.